src/main.js

```diff
@@ -9,13 +9,15 @@ var Menu = electron.Menu;
 var globalShortcut = electron.globalShortcut;
 var BrowserWindow = electron.BrowserWindow;
 
+const settings = require('electron-settings');
```

Comment:
Just realized my copy and paste from `electron-settings` docs resulted in a const. It works, but is not consistent with everything else...